engine/cluster/src/main/java/org/opennms/oce/engine/cluster/CEVertex.java

```diff
@@ -67,6 +69,11 @@ public ResourceKey getResourceKey() {
 
     public void addOrUpdateAlarm(Alarm alarm) {
         alarmsById.put(alarm.getId(), alarm);
+        updateTimestamp();
```

Comment:
Can we distinguish between the "created at" and "last updated at" timestamps? Would be useful to have both.